dotcom-rendering/src/lib/useCommentCount.ts

```diff
@@ -23,7 +26,9 @@ export const useCommentCount = (
 			.sort() // ensures identical sets produce the same query parameter
 			.join(','),
 	});
-	const url = `${discussionApiUrl}/getCommentCounts?${searchParams.toString()}`;
+	const url = `${upgradeToHttps(
```

Comment:
I'm not sure what the abstraction adds? Non-blocking.

Reply:
Reduces the creation of a regular expression on every invocation…